modules/mqtt/destination/Makefile.am

```diff
@@ -0,0 +1,31 @@
+if ENABLE_MQTT
+noinst_LTLIBRARIES      += modules/mqtt/destination/libmqtt_destination.la
+modules_mqtt_destination_libmqtt_destination_la_SOURCES = \
+  modules/mqtt/destination/mqtt-grammar.y       \
```

Comment:
the grammar/parser file should be common for both mqtt destination and a potential source, so they should IMHO be in the parent directory.

Reply:
also the mqtt-plugin.c file